doc/src/Run_options.rst

```diff
@@ -220,14 +221,32 @@ links with from the lib/message directory.  See the
 
 ----------
 
+.. _cite:
+
+**-cite style or file name**
+
+Select how and where to output a reminder about citing contributions
+to the LAMMPS code that were used during the run. Available styles are
+"both", "none", "screen", or "log". Any flag will be considered a file
```

Comment:
delete the Any flag ... sentence.  It's covered below.

Reply:
rewrote the text to remove some redundancy and made the second mention more to the point of why it was there.